locale/translations/en.json

```diff
@@ -81,6 +81,7 @@
     "error-url-not-table": "The URL is not associated with a table",
     "error-file-size-exceeds-10mb": "The total size of the files exceeds 10MB. This operation might take some time...",
     "checking-errors": "Checking errors",
+    "file-upload-dialog-description": "Here you can paste links from public Google Sheets and urls from csv files in open data portals and GitHub.",
```

Comment:
@romicolman 
I think this wording might be confusing as a like from GitHub won't actually work e.g. https://github.com/frictionlessdata/frictionless-py/blob/main/data/table.csv

"a URL pointing to a publicly available CSV file" (also might make sense to empathize that it needs to be the raw form of the file now a webview like on GitHub)